### PR TITLE
Create seed

### PR DESCRIPTION
**Pull request by schai6**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

Connected to #85 

Created seed file which populates watches, users, orders, reviews, and orderWatches.
Use by running `npm run seed`

### CODE REVIEW

File: seed.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+const chance = require('chance')(123);
+const db = require('./server/db')
+const { User, Watch, Review, Order, OrderWatch } = require('./server/db/models/')
+
+const watches = []
+const makes = ['Cartier', 'Laurent Ferrier', 'A. Lange & Söhne', 'Richard Mille', 'Baume & Mercier', 'Urwerk', 'Jaeger-LeCoultre', 'Parmigiani Fleurier', 'Panerai', 'Hermeès']
+const models = ['Drive de Cartier Extra-Flat', 'Galet Annual Calendar Montre École', 'Saxonia Triple Split', 'RM 53-01 Tourbillon Pablo Mac Donough', 'Clifton Club Legend Tributes', 'UR-210 Black Platinum', 'Polaris Memovox Limited Edition', 'Kalpa Hebdomadaire', 'Lo Scienziato  Luminor 1950 Tourbillon GMT Titanio', 'Carré H']
+const complicationOptions = ['Simple chronograph', 'Simple calendar',	'Alarm', 'Counter chronograph',	'Annual calendar', 'Quarter repeater', 'Split-second flyback chronograph',	'Perpetual calendar',	'Half-quarter repeater', 'Independent second-hand chronograph',	'Equation of time',	'Five-minute repeater', 'Jumping second-hand chronograph', 'Moon phases', 'Minute repeater']
+const minYear = 2000, maxYear = 2018
```
Comment (jluria) [inline seed.js]: Should we not have some older classic watches too? A wider range?

File: seed.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  Promise.all(watches.map(watch => Watch.create(watch)))
+  .then(() => Promise.all(users.map(user => User.create(user))))
+  .then(() => Promise.all(orders.map(order => Order.create(order))))
+  .then(() => Promise.all(reviews.map(review => Review.create(review))))
+  .then(() => Promise.all(orderWatches.map(orderWatch => OrderWatch.create(orderWatch))))
+
+const main = () => {
+  console.log('Syncing db...');
+  db.sync({
+      force: true
```
Comment (jluria) [inline seed.js]: This issues a DROP TABLE statement in SQL, right?

**Review by jluria — CHANGES_REQUESTED** [review]
Looks pretty good. 2 quick minor questions

File: seed.js (hunk 1 above)
Comment (schai6) [reply seed.js]: I don't think it makes a difference... but I changed the minYear for you haha.

File: seed.js (hunk 2 above)
Comment (schai6) [reply seed.js]: Yes it clears the database for reseeding.

**Review by jluria — APPROVED** [review]

**Pull request merged**